README.md

```diff
@@ -12,43 +12,100 @@
-The QuestDB SQL Web Console is an interactive SQL editor facilitating CSV import. Finally, QuestDB also includes the Postgres Wire Protocol
-for programmatic queries.
+> Ready to go? Jump to the
+> [Get started](#get-started) section.
 
-Popular tools that integrate with QuestDB include [Apache Kafka](https://questdb.io/docs/third-party-tools/kafka/questdb-kafka/), [Grafana](https://questdb.io/docs/third-party-tools/grafana/),
-[Superset](https://questdb.io/docs/third-party-tools/superset/), [Telegraf](https://questdb.io/docs/third-party-tools/telegraf/) and [Apache Flink](https://questdb.io/docs/third-party-tools/flink/).
+<p>&nbsp;</p>
 
 <div align="center">
-  <a href="https://demo.questdb.io">
-    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="600" />
+  <a href="https://demo.questdb.io/">
+    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="900" />
   </a>
+  <p><em>QuestDB Web Console - click to launch demo</em></p>
 </div>
 
-## Try QuestDB
+<p>&nbsp;</p>
+
+## Benefits of QuestDB
+
+QuestDB excels with:
+
+- financial market data
+- IoT sensors with high data cardinality
+- real-time dashboards
+
+Feature highlights include:
+
+- SQL with powerful, SIMD-optimized time-series extensions
+- High-speed ingestion via the InfluxDB Line Protocol
+- Strong and efficient performance on limited hardware
+- Columnar storage format (native or
+  [Apache Parquet](https://questdb.io/glossary/apache-parquet/)), partitioned
+  and ordered by time
+- Responsive and intuitive Web Console for query and data management, with error
+  handling
+- Excellent performance with
+  [high data cardinality](https://questdb.io/glossary/high-cardinality/) - see
+  [benchmarks](#questdb-performance-vs-other-oss-databases)
+
+And why use a time-series database?
 
-Our [live demo](https://demo.questdb.io/) is provisioned with the latest QuestDB release and sample datasets:
+Beyond performance and efficiency, with a specialized
+[time-series database](https://questdb.io/glossary/time-series-database/), you
+don't need to worry about:
+
+- out-of-order data
+- duplicates
+- exactly one semantics
+- streaming data (low latency)
+- high volumes of concurrent requests
+- volatile and "bursty" data
+- adding new columns - change schema "on the fly" while streaming data
+
+## Try QuestDB, demo and dashboards
+
+The [live, public demo](https://demo.questdb.io/) is provisioned with the latest
+QuestDB release and sample datasets:
 
 - Trips: 10 years of NYC taxi trips with 1.6 billion rows
 - Trades: live crypto market data with 30M+ rows per month
 - Pos: geolocations of 250k unique ships over time
 
+Use example queries or write your own!
+
+_The public demo queries over 1.6BN rows and uses 24 cores out
```

Comment:
We're using 46 shared workers, so it's more precise to say that we use 48 cores.
```suggestion
_The public demo queries over 1.6BN rows and uses 48 cores out
```